package.json

```diff
@@ -15,36 +15,44 @@
   "author": "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
   "contributors": [
     "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
-    "Christian Murphy <christian.murphy.42@gmail.com> (https://github.com/ChristianMurphy)"
+    "Christian Murphy <christian.murphy.42@gmail.com> (https://github.com/ChristianMurphy)",
+    "Thomas 'zemnmez' Shadwell (https://github.com/zemnmez)"
   ],
   "files": [
     "index.js"
   ],
   "dependencies": {
-    "unist-util-visit": "^1.0.0",
+    "unist-util-visit": "^2.0.0",
     "vfile-location": "^2.0.0"
```

Comment:
while it's not part of the public interface.
There may be value in holding off merging this until `vfile-location` gets types as well.
Bundling that major release update in with this one.

Reply:
https://github.com/vfile/vfile-location/pull/4

Reply:
Released!

Reply:
Thanks @wooorm!
Updated this PR with the major version update